doc/examples/widgets/lib/main.dart

```diff
@@ -108,8 +108,9 @@ void main() async {
           child: SpriteWidget(
             sprite: shieldSprite,
             anchor: parseAnchor(
-              ctx.listProperty('anchor', 'Anchor.center', anchorOptions),
-            ),
+                  ctx.listProperty('anchor', 'Anchor.center', anchorOptions),
+                ) ??
+                Anchor.topLeft,
```

Comment:
This isn't really necessary, the `Anchor.center` value on the listProperty method is already the default value, so there is no way this method would return null

Reply:
This is necessary because this addresses a possible `null` value returning from `parseAnchor`. See #607 which is extracted from here to change its logic in a separate changeset.

Reply:
Oh sorry, I missread this, I though the `??` was for `ctx.listProperty`, it is for `parseAnchor` so this makes sense.